samples/react-listview-context-ecb/src/webparts/emplopyeeListView/components/EmployeeListView.tsx

```diff
@@ -11,6 +11,8 @@ import { IECBProps } from './IECBProps';
 import { IEmployeeListViewProps } from './IEmployeeListViewProps';
 import { IEmployeeListViewState } from './IEmployeeListViewState';
 
+import { SPHpttClient, SPHttpClientResponse } from "@microsoft/sp-http";
```

Comment:
Typo here. Should be SPHttpClient. Also, import is not used because code is commented out.